### PR TITLE
Allow to reconfigure update_interval and send_status from web ui

### PR DESCRIPTION
**Pull request by mariussturm**

### CODE REVIEW

File: api/graylog.go [hunk 1]
```diff
@@ -130,9 +129,24 @@ func UpdateRegistration(httpClient *http.Client, context *context.Ctx, status *g
...
 		log.Error("[UpdateRegistration] Failed to report collector status to server: ", err)
 	}
+
+	// Update configuration based on server response
+	if respBody != nil {
+		// API query interval
+		if context.UserConfig.UpdateInterval != respBody.Configuration.UpdateInterval &&
+			respBody.Configuration.UpdateInterval > 0 {
+			log.Infof("[ConfigurationUpdate] update_interval: %ds", respBody.Configuration.UpdateInterval)
+			context.UserConfig.UpdateInterval = respBody.Configuration.UpdateInterval
```
Comment (bernd) [inline api/graylog.go]: Is this safe to modify from different go routines?

File: api/graylog.go [hunk 2]
```diff
@@ -130,9 +129,24 @@ func UpdateRegistration(httpClient *http.Client, context *context.Ctx, status *g
...
+		}
+		// Send host status
+		if context.UserConfig.SendStatus != respBody.Configuration.SendStatus {
+			log.Infof("[ConfigurationUpdate] send_status: %v", respBody.Configuration.SendStatus)
+			context.UserConfig.SendStatus = respBody.Configuration.SendStatus
```
Comment (bernd) [inline api/graylog.go]: See above.

File: api/graylog.go (hunk 1 above)
Comment (mariussturm) [inline api/graylog.go]: In this simple case it's safe I would say. `UpdateInterval` is initilized during startup by the config file. In the code we just update the value and dont rely on the exact order of the update. The worst case is that the old value will be used for a single next period.

**Comment by bernd** [comment]
LGTM :+1:

**Pull request merged**